### ISSUE TITLE
FileNotFoundException Not Very Explicit

### ISSUE THREAD

**Issue by zhuchcn** [uclahs-cds/pipeline-Nextflow-config#62]
https://github.com/uclahs-cds/pipeline-Nextflow-config/blob/f3448ac26989391df434a3d9c59a9227b136431d/config/schema/schema.config#L63

I found this error can be unclear sometimes. In my case, one of the files was given as an empty string. So All it prints out is:

```
ERROR ~ Unable to parse config file: 'metapipeline-DNA/module/recalibrate_BAM/./default.config'

   does not exist.
```

I had other cases that I had to spend a lot of time figuring out where the error came from. Maybe we can also print out what variable the path is missing.

**Comment by yashpatel6** [comment]
Agreed, it should be better, it'd need a slight update to the error handling to make sure the error message has the necessary information without having to pass it all to all of the functions
